doc/decoders.texi

```diff
@@ -461,6 +461,26 @@ The default is @var{true}.
 @item -canvas_size @var{image_size}
 Specify input video size to determine size of canvas to render subtitle.
 
+The aribcaption decoder assumes input frame size for bitmap rendering as bellow:
+@enumerate
+@item
+PROFILE_A : 1440 x 1080 with SAR 4:3
+@item
+PROFILE_C : 320 x 180 with SAR 1:1
+@end enumerate
+
+If actual frame aspect ratio (FAR) of input video does not match above assumption,
+the rendered captions may be distorted.
+To make the captions undistorted, add @code{-canvas_size} option to specify
+actual input video size.
```

Comment:
My terminology is based on following "mpeg4-avc terminology":
https://forum.videohelp.com/threads/323530-please-explain-SAR-DAR-PAR

To translate above to "MPEG2" terminology, _SAR_ is to be PAR, and _FAR_ (_input video size_) is tu be SAR. This may cause confusion.

Reply:
I drop the words "frame aspect ratio".
Instead, "frame size" is used.